### PR TITLE
feat: add telemetry infrastructure

### PR DESCRIPTION
**Pull request by nathanielc**
With these change metrics and traces are now collected.

Metrics can be viewed using Prometheus and traces via Jaeger.

### CODE REVIEW

File: k8s/opentelemetry/opentelemetry.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            - --config=/config/otel-config.yaml
+          ports:
+            - containerPort: 4317
+              name: otlp-receiver
+            - containerPort: 9090
+              name: prom-metrics
+            - containerPort: 8888
+              name: self-metrics
+          resources:
+            limits:
```
Comment (AaronGoldman) [inline k8s/opentelemetry/opentelemetry.yaml:35]: How did we pick these numbers?

**Review by AaronGoldman — APPROVED** [review]
Looks like a lot of moving parts but I don't see a way to have fewer of them.

File: k8s/ceramic/manifests/bootstrap.yaml [hunk 2]
```diff
@@ -16,15 +15,15 @@ spec:
           - "/usr/bin/runner"
           - "bootstrap"
         env:
+          - name: RUNNER_OTLP_ENDPOINT
+            value: 'http://otel:4317'
           - name: RUST_LOG
-            value: 'info,runner=debug'
+            value: 'info,runner=trace'
```
Comment (dbcfd) [inline k8s/ceramic/manifests/bootstrap.yaml:21]: Any reason why runner is at trace?

**Review by zachferland — APPROVED** [review]

File: k8s/ceramic/manifests/bootstrap.yaml (hunk 2 above)
Comment (nathanielc) [reply k8s/ceramic/manifests/bootstrap.yaml:21]: Runner is the code we write so figured it was nice to have full verbosity there. Obviously its easy to change.

**Pull request merged**